ca_gazebo/launch/prueba.launch

```diff
@@ -0,0 +1,29 @@
+<launch>
+	<arg name="env" default="prueba"/>
+
+  	<arg name="paused" value="$(optenv PAUSED false)"/>
+	<arg name="gui" value="$(optenv GUI true)"/>
+	<arg name="debug" value="$(optenv DEBUG false)"/>
+	
+	<env name="GAZEBO_MODEL_PATH" value="$(find ca_gazebo)/models"/>
```

Comment:
Replace this.

Reply:
This is an unnecessary file. Deleted it.